src/groups.js

```diff
@@ -0,0 +1,63 @@
+import Syncano from '@syncano/core';
+import { validateRequired, checkRequestMethodType } from './utils/helpers';
+
+export default async (ctx) => {
+  const { response, data } = new Syncano(ctx);
+  const { meta, args: { group_id, ...groupParams } } = ctx;
+  const { group_name, purpose } = groupParams;
```

Comment:
why not destructure in the line above?

Reply:
Since `purpose` param is optional, wanted to avoid passing `undefined` to the group db and also avoid multiple if statements before `creating` or `updating` a group. `const { group_name, purpose } = groupParams;` was to avoid things like `groupParams.group_name` since such is used frequently in file.

Reply:
This could be achieved with, i.e.:

`const { meta, args: { group_id, group_name, purpose = '' } } = ctx;`

Reply:
hmmm, we don't want to update the existing `purpose` field with `''` when updating